tshirts2022.py

```diff
@@ -0,0 +1,29 @@
+import json
+import re
+from urllib import request
+
+URL = "https://www.pycon.jp/support/bootcamp.html"
+with request.urlopen(URL) as f:
+    html = f.read().decode("utf-8")
+pattern = re.compile(r"pyconjp.*/event/(\d+)/(?!.*中止)")
+ids = ",".join(m[1] for m in pattern.finditer(html))
+
+connpass_api = "https://connpass.com/api/v1/event/"
+url = f"{connpass_api}?event_id={ids}&count=100"
```

Comment:
f-stringでURLパラメータ組み立てをやっていますが、それは狙い通り？
長くならないようにしている？

Reply:
これは長くならないためですね。ダサイけど

Reply:
微妙だけど、まぁいいかな。他の人の意見を聞きたい。

Reply:
https://docs.python.org/ja/3/library/urllib.request.html#urllib-examples  に則って

```suggestion
params = urllib.parse.urlencode({'event_id': ids, 'count': 100})
url = f"{connpass_api}?{params}"
```

あんまり良くなった感じはしないけど、公式で安全なやり方、という感じ

Reply:
これは長くなっちゃうので採用しない

Reply:
f-stringで違和感ないのでLGTMですー